<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportAlert</class>
 <widget class="QDialog" name="ExportAlert">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1351</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>优化建议</string>
  </property>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>530</y>
     <width>112</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>返回修改</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmButton">
   <property name="geometry">
    <rect>
     <x>811</x>
     <y>530</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>继续导出</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>731</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>标题</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>1061</width>
     <height>451</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="jumpUrlButton">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>20</y>
     <width>221</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>点击查看所有校验规则</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
